### PR TITLE
Update signing-macos.md

### PR DESCRIPTION
**Pull request by K-Codemagic**
Environmental groups updation for macOS code signing

### CODE REVIEW

**Comment by K-Codemagic** [comment]
Updated

File: content/yaml-code-signing/signing-macos.md [hunk 1]
```diff
@@ -49,17 +49,26 @@ Take note of the **Issuer ID** above the table of active keys as well as the **K
...
 environment:
-  vars:
-    APP_STORE_CONNECT_ISSUER_ID: Encrypted(...)
-    APP_STORE_CONNECT_KEY_IDENTIFIER: Encrypted(...)
-    APP_STORE_CONNECT_PRIVATE_KEY: Encrypted(...)
-    CERTIFICATE_PRIVATE_KEY: Encrypted(...)
+  groups:
+    - appstore_key_properties
+    
+  #Add the above mentioned group environment variables in Codemagic UI (either in Application/Team variables):
```
Comment (helinanever) [inline content/yaml-code-signing/signing-macos.md:59]:
```suggestion
  # Add the following environment variables in the Codemagic UI (either in Application or in Team variables): 
```

**Comment by helinanever** [comment]
It would make sense to organize the content as follows:
1) the environment variables that need to be added and how to save them in the UI
2) how to export all the variables to yaml

File: content/yaml-code-signing/signing-macos.md [hunk 2]
```diff
@@ -70,11 +61,11 @@ environment:
 
 - `APP_STORE_CONNECT_PRIVATE_KEY`
 
-  This is the private API key downloaded from App Store Connect. You'll need to [encrypt](../building/encrypting/) the **contents** of the file in the codemagic.yaml editor UI (and not the file itself). On macOS, you can use `pbcopy < AuthKey_XXXXXX.p8` to copy the contents of the private key and paste this into the UI encryption tool.
+  This is the private API key downloaded from App Store Connect. On macOS, you can use `pbcopy < AuthKey_XXXXXX.p8` to copy.
```
Comment (helinanever) [inline content/yaml-code-signing/signing-macos.md:64]: is there a part of instructions missing here?

File: content/yaml-code-signing/signing-macos.md [hunk 3]
```diff
@@ -127,16 +139,20 @@ Instead of specifying the exact bundle-id, you can use `"$(xcode-project detect-
 
 ## Provide signing files manually
 
-In order to use manual code signing, [encrypt](../building/encrypting/#encrypting-sensitive-data) your **signing certificate**, the **certificate password** (if the certificate is password-protected) and the **provisioning profile**, and set the encrypted values to the following environment variables. Note that to encrypt files, they will have to be base64 encoded and will have to be decoded during the build.
+In order to use manual code signing, Save your **signing certificate**, the **certificate password** (if the certificate is password-protected) and the **provisioning profile** in the **Environment variables** section in Codemagic UI. Select **Secure** to encrypt the values. Note that binary files (i.e. provisioning profiles & .p12 certificate) have to be [`base64 encoded`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) locally before they can be saved to **Environment variables** and decoded during the build.
```
Comment (helinanever) [inline content/yaml-code-signing/signing-macos.md:142]:
```suggestion
In order to use manual code signing, save your **signing certificate**, the **certificate password** (if the certificate is password-protected) and the **provisioning profile** in the **Environment variables** section in Codemagic UI. Select **Secure** to encrypt the values. Note that binary files (i.e. provisioning profiles & .p12 certificate) have to be [`base64 encoded`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) locally before they can be saved to **Environment variables** and decoded during the build.
```

File: content/yaml-code-signing/signing-macos.md (hunk 2 above)
Comment (K-Codemagic) [reply content/yaml-code-signing/signing-macos.md:64]: no, I think it getting repeated over and over again to secure the variables. It is already mentioned above.

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**